config/manager/manager.yaml

```diff
@@ -3,7 +3,6 @@ apiVersion: apps/v1
 kind: Deployment
 metadata:
   name: tf-controller
-  namespace: flux-system
```

Comment:
I moved `namespace` to the `kustomization.yaml` so that `make deploy` can install tf-controller in `tf-system` namespace while still keeping the deployment release manifest to have `flux-system` as its namespace.